src/Graphics/src/Graphics/Platforms/iOS/PlatformGraphicsView.cs

```diff
@@ -8,9 +8,9 @@ namespace Microsoft.Maui.Graphics.Platform
 	[Register(nameof(PlatformGraphicsView))]
 	public class PlatformGraphicsView : UIView
 	{
-		private WeakReference<IGraphicsRenderer> _renderer;
+		private IGraphicsRenderer _renderer;
 		private CGColorSpace _colorSpace;
-		private WeakReference<IDrawable> _drawable;
+		private IDrawable _drawable;
```

Comment:
If this test case still passes, this is probably good 👍 :

https://github.com/dotnet/maui/blob/be8bda69153fb8f51712ac024fea40e14644e1b4/src/Controls/tests/DeviceTests/Memory/MemoryTests.cs#L59

There may also be other cases that depend on `PlatformGraphicsView`: `Polygon`? `Polyline`?

Reply:
yea I ran the whole set of "leak" tests locally and they all passed